src/GovukNotify/GovukNotify.csproj

```diff
@@ -16,11 +16,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="JWT" Version="[7.1,9)" />
+    <PackageReference Include="JWT" Version="[7.1,10)" />
```

Comment:
Hi @RocktimeLtd - in your issue you asked for a version that will accept the latest version of JWT, "currently 10.0.2". This patch looks like it excludes v10 - is that right? Should this be `[7.1,11]`?

Reply:
Hi Samuel,

You were completely correct. I had miss typed the version string. I have changed it to [7.1,11)
er 7.1 >= x < 11

Regards Kieron